pkg/cmd/server/start/command_test.go

```diff
@@ -338,36 +277,26 @@ func executeAllInOneCommandWithConfigs(args []string) (*MasterArgs, *configapi.M
 	root.SetArgs(argsToUse)
 	root.Execute()
 
-	masterCfg, masterErr := configapilatest.ReadAndResolveMasterConfig(fakeMasterConfigFile.Name())
-	nodeCfg, nodeErr := configapilatest.ReadAndResolveNodeConfig(fakeNodeConfigFile.Name())
-
-	return cfg.MasterArgs, masterCfg, masterErr, cfg.NodeArgs, nodeCfg, nodeErr
-}
-
-func executeNodeCommand(args []string) *NodeArgs {
-	fakeConfigFile, _ := ioutil.TempFile("", "")
-	defer os.Remove(fakeConfigFile.Name())
-
-	argsToUse := make([]string, 0, 4+len(args))
-	argsToUse = append(argsToUse, "node")
-	argsToUse = append(argsToUse, args...)
-	argsToUse = append(argsToUse, "--write-config")
-	argsToUse = append(argsToUse, "--create-certs=false")
-	argsToUse = append(argsToUse, "--config="+fakeConfigFile.Name())
-
-	root := &cobra.Command{
-		Use:   "openshift",
-		Short: "test",
-		Long:  "",
-		Run: func(c *cobra.Command, args []string) {
-			c.Help()
-		},
+	masterCfg, masterErr := configapilatest.ReadAndResolveMasterConfig(path.Join(getAllInOneConfigDir(), "master", "master.yaml"))
+
+	var nodeCfg *configapi.NodeConfig
+	var nodeErr error
+
+	dir, nodeErr := os.Open(getAllInOneConfigDir())
```

Comment:
use http://golang.org/pkg/path/filepath/#Glob to search `[dir]/node-*/node-config.yaml` instead of looping. We should probably only have a single one?